kotlinx.fuzz.jazzer/src/main/kotlin/kotlinx/fuzz/jazzer/JazzerEngine.kt

```diff
@@ -35,16 +41,40 @@ class JazzerEngine(private val config: KFuzzConfig) : KFuzzEngine {
         config.exceptionsDir.createDirectories()
     }
 
-    @OptIn(ExperimentalPathApi::class)
+    private fun isDebugMode(): Boolean = ManagementFactory.getRuntimeMXBean().inputArguments.any { it.contains("-agentlib:jdwp") }
+
+    private fun getDebugSetup(intellijDebuggerDispatchPort: Int, method: Method): List<String> {
+        val port = ServerSocket(0).use { it.localPort }
+        Socket("127.0.0.1", intellijDebuggerDispatchPort).use { socket ->
+            DataOutputStream(socket.getOutputStream()).use { output ->
+                output.writeUTF("Gradle JVM")
+                output.writeUTF("Jazzer Run Target: ${method.name}")
+                output.writeUTF("DEBUG_SERVER_PORT=$port")
```

Comment:
where is this text printed to? didn't see it in any logs, what is its purpose?

Reply:
It is not printed anywhere it is needed for requesting debug server (as written [here](https://jetbrains.team/blog/DX%3B_Deep_Debugging_and_my_new_favorite_System_Property))